js/background.js

```diff
@@ -77,7 +77,8 @@
     theme: 'sunburst',
     font: 'Inconsolata',
     fontSize: 'medium',
-    lineNumbers: true
+    lineNumbers: true,
+    languageBlacklist: ''
```

Comment:
Yes, you're right. Without wanting to overly complicate things, for this PR in particular, but allowing for domain blacklisting to gracefully extend this work, what would you think of:

```js
blacklists: {
    languages: ['javascript', 'ruby'],
    domains: ['github.com']
}
```